activerecord/lib/active_record/schema_migration.rb

```diff
@@ -31,8 +35,10 @@ def create_table(limit=nil)
 
           connection.create_table(table_name, id: false) do |t|
             t.column :version, :string, version_options
+            t.column :created_at, :datetime, null: false
```

Comment:
I have attempted to make similar change, and one difference that I thought about is doing an `add_column` in case the table already exist. so existing app would get the new column...
i don't know if you thought of something else for existing app upgrading rails to a version that would have this feature ?
https://github.com/rails/rails/pull/30289/files#diff-a15c218db8b9dd9f84a75d9a3894db56R26